user-lock-manager-w-keypad-support.smartapp.groovy

```diff
@@ -1066,20 +1092,29 @@ def performActions(evt) {
     def codeData = new JsonSlurper().parseText(evt.data)
     if(enabledUsersArray().contains(codeData.usedCode) || isManualUnlock(codeData)) {
       // Global Hello Home
-      if (noRunPresence && doRunPresence == null) {
-        if (!anyoneHome(noRunPresence)) {
-          location.helloHome.execute(homePhrases)
-        }
-      } else if (doRunPresence && noRunPresence == null) {
-        if (anyoneHome(doRunPresence)) {
-          location.helloHome.execute(homePhrases)
-        }
-      } else if (doRunPresence && noRunPresence) {
-        if (anyoneHome(doRunPresence) && !anyoneHome(noRunPresence)) {
-          location.helloHome.execute(homePhrases)
-        }
-      } else {
-       location.helloHome.execute(homePhrases)
+      if(location.currentMode != homePhrases)
```

Comment:
Would you mind keeping with the code style of the rest of the document?  example:

```
if (something) {
 //do some things
}
```

instead of:
```
if (something)
{
  //do something else
}
```